core/misc/autosubmit.js

```diff
@@ -37,12 +37,39 @@
 Backdrop.behaviors.autosubmit = {
   attach: function(context) {
     // 'this' references the form element
-    function triggerSubmit (e) {
+    function triggerSubmit (element) {
       var $this = $(this);
+      var $triggering_element = $(element);
+
+      // Variable "element" will have a value only when text fields trigger
+      // this. If element is undefined, then remove the data to prevent
+      // potential focus on a previously processed element.
+      if (element === undefined)  {
+        $('body').removeData('needs-focus');
+      }
+      else if ($triggering_element.parents('.view').length) {
```

Comment:
I don't think we want to put a Views-specific selector in here. I don't think there would be harm in just always populating this data attribute and always re-focusing the last autocomplete element after autosubmit.